independent-projects/tools/codestarts/src/main/java/io/quarkus/devtools/codestarts/Codestart.java

```diff
@@ -1,6 +1,7 @@
 package io.quarkus.devtools.codestarts;
 
-import java.util.Comparator;
+import io.quarkus.devtools.codestarts.core.CodestartSpec;
+import io.quarkus.devtools.codestarts.utils.NestedMaps;
```

Comment:
Is this a leftover?

Reply:
@aloubyansky what do you mean?

Reply:
the comparator are now in `DefaultCodestartProjectDefinition`

Reply:
Are these imports necessary?

Reply:
Ah ok @aloubyansky , they were in the same package :)